SparkPost/2web2ui
# FE-789 Cypress tests for Summary Report

### What Changed
 - Added integration tests for Summary Report.
    - Changing date range
    - Adding filters
    - Changing metrics
    - Changing group by(see notes)


### How To Test
 - run `npm run test-integration` and `npm run test-integration-headless`
 - Uncomment out the 4th test and confirm that it works locally.

### Notes
 - I commented out one test because it was working locally but failing on travis
